user_test.go

```diff
@@ -83,12 +84,23 @@ type TestIdentityRepository struct {
 	Identity *account.Identity
 }
 
+// List returns a list identities
+func (m *TestIdentityRepository) List(ctx context.Context) (*app.IdentityArray, error) {
+	res := app.IdentityArray{}
+	res.Data = make([]*app.IdentityData, 1)
+	i := account.Identity{FullName: "Test user", ImageURL: "http://a.com"}
+	ident := i.ConvertIdentityFromModel()
+	res.Data[0] = ident.Data
+
+	return &res, nil
+}
+
 // Load returns a single Identity as a Database Model
-func (m *TestIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*account.Identity, error) {
+func (m *TestIdentityRepository) Load(ctx context.Context, id string) (*app.Identity, error) {
 	if m.Identity == nil {
 		return nil, errors.New("not found")
 	}
-	return m.Identity, nil
+	return m.Identity.ConvertIdentityFromModel(), nil
```

Comment:
the account.Identity shouldn't know about the external API it's being exposed as. I can have multiple external representations.